mne/preprocessing/maxwell.py

```diff
@@ -41,6 +41,88 @@
 # differences between algorithms
 
 
+@verbose
+def maxwell_filter_prepare_emptyroom(
```

Comment:
I would rather just call this `prepare_emptyroom`, by living in `mne.preprocessing.maxwell.prepare_emptyroom` its clear it's for MF already (and I like the mne.preprocessing.maxwell scope better anyway)

Reply:
> I would rather just call this `prepare_emptyroom`, by living in `mne.preprocessing.maxwell.prepare_emptyroom` its clear it's for MF already (and I like the mne.preprocessing.maxwell scope better anyway)

The problem with this is that we've trained our users to look for `maxwell_filter` in `mne.preprocessing`, not in `mne.preprocessing.maxwell`. The change of namespace would cause confusion

Reply:
I agree that it's harder to discover interactively with what @larsoner suggests